frontend/__mocks__/mdmMock.ts

```diff
@@ -79,3 +80,17 @@ export const createMockHostMdmData = (
 ): IHostMdmData => {
   return { ...DEFAULT_HOST_MDM_DATA, ...overrides };
 };
+
+const DEFAULT_SETUP_EXPIERENCE_SCRIPT: IGetSetupExperienceScriptResponse = {
+  id: 1,
+  team_id: 0,
```

Comment:
```suggestion
  team_id: null,
```

On the backend, this has the same shape as a normal script. For no team scripts, the team id is null in API responses.

Reply:
udpated